Sources/BraveWallet/Crypto/Stores/PortfolioStore.swift

```diff
@@ -33,6 +43,8 @@ public class PortfolioStore: ObservableObject {
   @Published private(set) var balance: String = "$0.00"
   /// The users visible assets
   @Published private(set) var userVisibleAssets: [AssetViewModel] = []
```

Comment:
probably need to update the comments to be visible fungible tokens

Reply:
Updated in [9cd1e19](https://github.com/brave/brave-ios/pull/6380/commits/9cd1e19be73e66b2650137a5a1b088809e835dde)